modepy/quadrature/jacobi_gauss.py

```diff
@@ -261,9 +290,94 @@ def legendre_gauss_lobatto_nodes(
         backend: Optional[str] = None,
         force_dim_axis: bool = False) -> np.ndarray:
     """Compute the Legendre-Gauss-Lobatto quadrature nodes.
-    *N* is the number of points.
+    *N+1* is the number of nodes.
 
     Exact to degree :math:`2N - 1`.
     """
     return jacobi_gauss_lobatto_nodes(0, 0, N,
             backend=backend, force_dim_axis=force_dim_axis)
+
+
+class JacobiGaussLobattoQuadrature(Quadrature):
+    r"""
+    Compute the Jacobi-Gauss-Lobatto quadrature with respect
+    to the Jacobi weight function
+
+    .. math::
+
+        I[f] = \int_{-1}^1 f(x) (1 - x)^\alpha (1 + x)^\beta\, \mathrm{d}x,
+
+    There will be *N+1* nodes. Exact to degree :math:`2N - 3`.
+
+    .. versionadded:: 2024.2
+    """
+
+    def __init__(self,
+            alpha: float, beta: float, N: int,  # noqa: N803
+            *, backend: Optional[str] = None,
+            force_dim_axis: bool = False) -> None:
+        nodes = jacobi_gauss_lobatto_nodes(alpha, beta, N, backend)
+
+        from math import gamma
+
+        from modepy.modes import binom, scaled_jacobi as jacobi
```

Comment:
```suggestion
        from modepy.modes import binom, scaled_jacobi
```
? This was confusing :(

Reply:
Done! Sorry.